action.yml

```diff
@@ -34,8 +34,8 @@ runs:
       id: action
       shell: powershell
       run: |
-        $service_credential_password = ConvertTo-SecureString -String "${{ inputs.service-credential-password }}" -AsPlainText -Force
-        $secure_password = ConvertTo-SecureString -String "${{ inputs.service-account-password }}" -AsPlainText -Force
+        $service_credential_password = ConvertTo-SecureString -String '${{ inputs.service-credential-password }}' -AsPlainText -Force
+        $secure_password = ConvertTo-SecureString -String '${{ inputs.service-account-password }}' -AsPlainText -Force
```

Comment:
Still won't work if the passwords contain a `'` character.

Reply:
If we can convince the raw input values into the values of environment variables, the PowerShell script can access the values from environment variables without quoting.

```
    - name: Action
      id: action
      shell: powershell
      env:
        SERVICE_CRED_PW: inputs.service-credential-password
        SERVICE_ACCOUNT_PW: inputs.service-account-password
      run: |
        $service_credential_password = ConvertTo-SecureString -String $Env:SERVICE_CRED_PW -AsPlainText -Force
        $secure_password = ConvertTo-SecureString -String $Env:SERVICE_ACCOUNT_PW -AsPlainText -Force
```

Reply:
I tested a version of this with `deploy-windows-files` and it still gives the same error when a quote is one of the symbol characters.  I did find a fix using `@'` to encapsulate the string.